README.md

```diff
@@ -16,7 +16,7 @@ Link (for React Native lower than 0.60)
 
 ## Android
 
-See https://github.com/TimoGlastra/ExampleRnIndySDK for an example android react native project that follows this tutorial.
+See https://github.com/TheTreek/rn-indy-demo for an example android react native project that follows this tutorial.
```

Comment:
This repo uses a custom packaged version of rn-indy-sdk (and also still uses rn-indy-sdk). Do you think it would be possible to update `TimoGlastra/ExampleRnIndySDK` to the latest `indy-sdk-react-native` version?